jOOQ/src/main/java/org/jooq/impl/ExecuteListeners.java

```diff
@@ -102,7 +102,15 @@ static ExecuteListener get(ExecuteContext ctx) {
             // [#6747] Avoid allocating the listener (and by consequence, the ExecuteListeners) if
             //         we do not DEBUG log anyway.
             if (LOGGER_LISTENER_LOGGER.isDebugEnabled())
-                (result = init(result)).add(new LoggerListener());
+                (result = init(result)).add(new LoggerListener(
+                        ctx.settings().getExecuteLoggingBindVariables(),
+                        ctx.settings().isExecuteLoggingAbbreviatedBindVariables(),
+                        ctx.settings().getExecuteLoggingSqlString(),
+                        ctx.settings().getExecuteLoggingResult(),
+                        ctx.settings().getExecuteLoggingResultNumberOfRows(),
+                        ctx.settings().getExecuteLoggingResultNumberOfColumns(),
+                        ctx.settings().getExecuteLoggingRoutine(),
+                        ctx.settings().getExecuteLoggingException()));
```

Comment:
Does the logger listener really need to be aware of all of these at instantiation time? It will get a reference to `Settings` via the `ExecuteContext.settings()` method, so I don't see the point of this particular change...